quickget

```diff
@@ -422,7 +422,7 @@ function releases_opensuse(){
 }
 
 function releases_oraclelinux() {
-    echo 7.7 7.8 7.9 8.2 8.3 8.4 8.5
+    echo 7.7 7.8 7.9 8.4 8.5 8.6 9.0
```

Comment:
8.2 and 8.3 still exist and can be referenced but I think we should follow their lead since they no longer reference them on the download pages.